src/utils/responseHandlers.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @param {object} res - response object from the controller
+ * @param {string} code - The http status code to be sent
+ * @param {string} message - The message to send to the user
+ * @param {object} data - contains the details of the response if any (optional)
+ */
+const resSuccess = (code, message, data, res) => {
+  const body = {
+    status: 'success',
+    message,
+    data: data || '',
```

Comment:
@kelue If you don't pass any data it will return string, set it to empty object by default